Amasty_Storelocator/templates/pages/reviews.phtml

```diff
@@ -0,0 +1,287 @@
+<?php
+
+/**
+ * @author Amasty Team
+ * @copyright Copyright (c) 2020 Amasty (https://www.amasty.com)
+ * @package Amasty_Storelocator
+ */
+?>
+<?php
+/**
+ * @var \Amasty\Storelocator\Block\View\Reviews $block
+ * @var \Amasty\Storelocator\Model\Location $location
+ */
+$location = $block->getLocation();
+if ($location && $block->isReviewsEnabled()) :
+    $locationReviews = $location->getLocationReviews();
+    $countOfRatings =  $locationReviews ? count($locationReviews) : 0;
+    $format = $location->getDateFormat() ?: \IntlDateFormatter::MEDIUM;
+    $locationAvgRating = ($location->getLocationAverageRating())
+        ? $location->getLocationAverageRating()
+        : 0;
+?>
+    <div class="amlocator-block -separator">
+        <hr class="hr" />
+    </div>
+    <div
+        class="
+            amlocator-block
+            -reviews
+            amlocator-reviews-wrapper
+        "
+    >
+        <div
+            class="amlocator-collapse-title"
+            data-amlocator-js="collapse-trigger"
+        >
+            <span>
+                <?= $block->escapeHtml(__('Reviews')); ?>
+            </span>
+            <span class="amlocator-arrow" data-amlocator-js="collapse-indicator"></span>
+        </div>
+
+        <div class="amlocator-reviews-general">
+            <span class="amlocator-value">
+                <?= (float)round($locationAvgRating / 20, 1); ?>
+            </span>
+            <div class="amlocator-wrapper">
+                <div
+                    class="amstars-rating-container -small"
+                    title="<?= (float)$locationAvgRating ?>%"
```

Comment:
can this be escaped? if not, please add the comment:
`/** @escapeNotVerified */`